project1/CLI_strings/main.c

```diff
@@ -0,0 +1,35 @@
+#include "main.h"
+
+
+int main(int argc, char *argv[]) {
+
+	///*handling main arguments*/
+	if (argc != 3) {
+		printf("Wrong number of arguments, expected 2\n");
+		return EXIT_FAILURE;
+	}
+	FILE* file;
+	/*variables declaraion*/
+	char read_buff[BUFF_SIZE];
+
+	/*getting file to program's memory*/
+	if ((file = fopen(argv[1], "r")) == NULL) {
+		printf("File opening error!\n");
```

Comment:
**suggestion**: You can improve the error message by printing the path to the invalid/non-existing file.